nationalarchives/ds-wagtail
# Styles for the non-js version of the what's on listing filters

[Ticket URL: paste URL here](https://national-archives.atlassian.net/wiki/spaces/DSWhatson/pages/151420936/What+s+on+listing+filters+spec)

Design: https://www.figma.com/proto/pcIxOpwvJ1YRYbt9lbeKUG/TNA-Etna---Design-system?node-id=5218-74239

## About these changes
These changes add the filter styles for the what's on listing, along with the pink header area
The styles for the breadcrumb, title and intro follow existing patterns in the site rather than following the design.

As we agreed not to use a detail / summary element for the dropdowns for the non-js version, the design has been adapted (in conversation with Nick)

The content is all hard-coded for now.

## How to check these changes
- Check out the `feature/filters-front-end` branch
- Create a 'what's on listing' page in your local build
- Review the styles

## Screenshots
<details>
<summary>click to expand</summary>

Desktop:
<img width="1437" alt="Screenshot 2023-10-03 at 15 12 36" src="https://github.com/nationalarchives/ds-wagtail/assets/771869/87d8a348-7ec3-4479-a8a3-5fbdf6ff50a8">

Mobile:
<img width="498" alt="Screenshot 2023-10-03 at 15 12 57" src="https://github.com/nationalarchives/ds-wagtail/assets/771869/1b4b6fe3-38e7-47cb-82c4-62b696da57b0">

</details>

## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
